src/main/java/io/split/android/client/SplitClient.java

```diff
@@ -73,4 +73,56 @@
     boolean isReady();
 
     void on(SplitEvent event, SplitEventTask task);
+
+    /**
+     * Enqueue a new event to be sent to split data collection services
+     *
+     * Example:
+     *      client.track(“account”, “checkout”)
+     *
+     * @param trafficType the type of the event
+     * @param eventType the type of the event
+     *
+     * @return true if the track was successful, false otherwise
+     */
+    boolean track(String trafficType, String eventType);
+
+    /**
+     * Enqueue a new event to be sent to split data collection services
+     *
+     * Example:
+     *      client.track(“account”, “checkout”, 200.00)
+     *
+     * @param trafficType the type of the event
+     * @param eventType the type of the event
+     * @param value the value of the event
+     *
+     * @return true if the track was successful, false otherwise
+     */
+    boolean track(String trafficType, String eventType, double value);
+
+    /**
+     * Enqueue a new event to be sent to split data collection services
+     *
```

Comment:
Add documentation explaning that the traffic type is the one defined whle constructing the client